app/code/community/Reverb/ReverbSync/Helper/Orders/Creation/Address.php

```diff
@@ -22,8 +26,10 @@ public function addOrderAddressAsShippingAndBillingToQuote(stdClass $reverbOrder
         $shippingAddressObject = $reverbOrderObject->shipping_address;
         if (!is_object($shippingAddressObject))
         {
-            $error_message = $this->__(self::ERROR_NO_ADDRESS);
-            throw new Exception($error_message);
+            // In this event, we still want to create the Magento order. We will populate the address fields with a
+            //      fake placeholder value
+            $this->_addNoAddressToOrderObject($reverbOrderObject);
+            $shippingAddressObject = $reverbOrderObject->shipping_address;
```

Comment:
@skwp we should have plugged that hole for these addresses. When you check out, we now check that you have a country code, even for people with current addresses on file.

Not saying that it can't happen, but we did a lot of work making sure that it doesn't.